src/components/test-results/test-results.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactEcharts from "echarts-for-react";
+import CaretSvg from "../../images/icons/caret-down.svg";
+import "./test-results.styles.scss";
+import { Label } from "../label/label";
+import format from "date-fns/format";
+import fromUnixTime from "date-fns/fromUnixTime";
+
+export const TestResults = ({ personalData = {}, results = [] }) => {
+  const xAxisData = results.map(item =>
+    format(fromUnixTime(item.date), "dd.MM.yyyy / HH:mm")
+  );
+  const yAxisData = results.map(item => ({
+    value: item.temperature,
+    itemStyle: { color: item.temperature > 37 ? "red" : "blue" }
+  }));
+
+  console.log(yAxisData);
```

Comment:
We might need to add an es-lint rule to not allow `console.log`s

In the meantime, could you please remove it 😁